test/test_test.go

```diff
@@ -16,6 +16,7 @@ package ciliumTest
 
 import (
 	// test sources
+	_ "github.com/cilium/cilium/test/benchmark"
 	_ "github.com/cilium/cilium/test/k8sT"
 	_ "github.com/cilium/cilium/test/runtime"
 )
```

Comment:
There may need to be more work done to handle this new `Benchmark` scope - cc: @eloycoto .

Reply:
I think that the scope shouldn't need updating unless we create a new topology. Spoke offline with @jrfastab about this and he's been trying this with Runtime scope.